tests/protractor/e2e/contact-summary.js

```diff
@@ -144,15 +145,17 @@ describe('Contact summary info', () => {
     }, 10000);
   };
 
-  it('contact summary', () => { //disabled.
-    helper.clickElement(element(by.css('#contacts-tab')));
+  it('contact summary', () => { 
     try {
+      helper.clickElement(element(by.css('#contacts-tab')));
+      
       selectContact('carol');
     }
     catch (err) {
       browser.refresh();
-      browser.sleep(500);//wait for browser to settle
-      helper.clickElement(element(by.css('#contacts-tab')));
+      browser.sleep(500);//wait for browser to settle      
+      common.goToPeople();
+      utils.handleModal();
```

Comment:
If we have to catch it here then we're not handling the modal correctly. This needs to be done in the function that changes the settings or it's going to come up in the middle of all sorts of tests. How did a modal happen here?